DeDRM_plugin/adobekey.py

```diff
@@ -110,15 +117,15 @@ def unicode_argv():
             # Remove Python executable and commands if present
             start = argc.value - len(sys.argv)
             return [argv[i] for i in
-                    xrange(start, argc.value)]
+                    range(start, argc.value)]
         # if we don't have any arguments at all, just pass back script name
         # this should never happen
         return [u"adobekey.py"]
     else:
         argvencoding = sys.stdin.encoding
```

Comment:
```suggestion
        argvencoding = sys.stdin.encoding or "utf-8"
```
and remove the next two lines because all falsey values are just as dangerous as None.